duckduckgo/iOS
# Feature/tab persistence

Reviewer: Caine
Asana: https://app.asana.com/0/361428290920652/377272475937399

**Description**:
Split out tab manager and tab model logic to allow for persistence. Added simple tab persistence.

**Steps to test this PR**:
- Open app
- open some tabs
- move around
- kill and reopen apps - tabs should still be there

Note: The full navigation stack could not be persisted (just the current page) due to webkit limitations (navigation is read only so there is no way to set it). In the future we could create our own nav stack.

###### Reviewer Checklist:


###### PR DRI Checklist:
